cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,6 @@
+describe('home page', () => {
+  it('the h1 contains the correct text', () => {
+    cy.visit('https://work.co/')
+    cy.get('h1').contains("We solve complex problems through design & technology")
```

Comment:
not related to this challenge, but since you have the fixture by default, you can edit the file and add this key, value => 'h1':"We solve complex problems through design & technology", and then instead of using hardcoded strings for validation, use it from fixture, check it out here https://docs.cypress.io/api/commands/fixture